environment/commands/start/index.js

```diff
@@ -1,7 +1,189 @@
-const { RainbowRunner } = require('../../lib/runner');
+const util = require('util');
+const {execSync} = require('child_process');
+const fs = require('fs');
+const os = require('os');
+const path = require('path');
+const request = require('request');
+
+const ProcessManager = require('pm2');
+
+const GANACHE_PORT = 9545;
+const NEAR_PORT = 3030;
 
 class StartCommand {
-    static async execute (service) { await (new RainbowRunner()).run(service); }
+  static getGanacheConfig(port = GANACHE_PORT) {
+    let accounts = "";
+    let accountPrefix =
+        "x2bdd21761a483f71054e14f5b827213567971c676928d9a1808cbfa4b750120";
+
+    for (let i = 0; i < 10; i++) {
+      accounts += util.format(
+          "--account=%s%d,1000000000000000000000000 ",
```

Comment:
Will `%d` add a leading zero when i<10?

Reply:
I think it will not. we want `--account=prefix 1,1000000000000000000000000` here?

Reply:
We want `prefix01` here.

Reply:
Got it